src/tribol/interface/tribol.cpp

```diff
@@ -961,17 +1024,19 @@ integer update( integer cycle, real t, real &dt )
 
       CouplingScheme* couplingScheme  = csManager.getCoupling(csIndex);
 
+      // TODO verify that all validity logging is called by all ranks
       // initialize and check for valid coupling scheme
       if (!couplingScheme->init())
       {
+         // TODO figure out if we want to call couplingScheme->apply() for null meshes
          if (couplingScheme->nullMeshes())
          {
             continue;
          }
          else
          {
-            SLIC_WARNING("Skipping invalid CouplingScheme " << couplingScheme->getId() << ". " << 
-                         "Please see warnings.");
+            SLIC_INFO("tribol::update(): skipping invalid CouplingScheme " << 
+                      couplingScheme->getId() << "Please see warnings.");
```

Comment:
Please explain why this is no longer a warning -- is it b/c it's expected behavior for some user codes?

Reply:
Good catch. This should be a warning. Fixed. 

I updated this to be `SLIC_WARNING_ROOT'. If a coupling scheme is invalid on one rank for non-null meshes it should be invalid on all ranks for non-null meshes.